modules/olm-why-use-operators.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * operators/what-are-operators.adoc
+
+[id='olm-why-use-operators-{context}']
+= Why use Operators?
+
+Operators provide:
+
+--
+- Repeatability of installation and upgrade.
+- Constant health checks of every system component.
+- Over-the-air (OTA) updates for OpenShift components and ISV content.
+- A place to encapsulate knowledge from field engineers and spread it to all
+users, not just one or two.
+--
+
+Why deploy on Kubernetes?::
+Kubernetes (and by extension, {product-title}) contains all of the primitives
+needed to build complex distributed systems – secret handling, load balancing,
+service discovery, autoscaling – that work across on-premise and cloud
+providers.
+
+Why manage your app with Kubernetes APIs and `kubectl` tooling?::
+These APIs are feature rich, have clients for all platforms and plug into the
+cluster’s access control/auditing. An Operator uses the Kubernetes' extension
+mechanism, Custom Resource Definitions (CRDs), so your custom object, e.g.
```

Comment:
Do you guys try to avoid e.g.? Like using "for example" instead as recommended by the IBM style guide?

Reply:
Yes, to my dismay. :disappointed:

Reply:
I've updated the "i.e."s and "e.g."s except for the ones in the CLI reference tables. ... :no_mouth: